P202.Training.WCF/P202.Training.WCF.csproj

```diff
@@ -30,6 +31,8 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <CodeAnalysisRuleSet>..\..\..\..\Downloads\CSharpStyleguide.ruleset</CodeAnalysisRuleSet>
```

Comment:
I think you're making reference to a path in your local disk that is not present in the repository. It's ok if you want to add this ruleset file in the root of the project and make a reference to it